server/core/statistics.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package core
+
+import (
+	"github.com/montanaflynn/stats"
+)
+
+// RollingStats provides rolling statistics with specified windows size.
+// There have windows size records used for calculating. it provides Rolling
+// `Median` now, which used to filter noise in some situation.
```

Comment:
Hard for me to understand... @queenypingcap Could you help to make the comment more fluent?

Reply:
How about the following?
RollingStats provides rolling statistics with specified window size. There are window size records for calculating. It provides median of the statistics to filter noises in some cases.